src/module/Ui/templates/editor/layout.phtml

```diff
@@ -54,10 +57,7 @@ echo $this->doctype();
 </div>
 
 <?php echo $this->partial('layout/partials/default-modal'); ?>
-<?php echo $this->headScript(); ?>
-<?php echo $this->partial('layout/partials/mathjax'); ?>
-<?php echo $this->inlineScript(); ?>
```

Comment:
You removed `inlineScript` everywhere, what did it do, exactly?

Reply:
Same as `headScript`